app/design/frontend/base/default/template/razorpay/placeorder.phtml

```diff
@@ -25,9 +26,16 @@
     };
 
     var placeOrder = function() {
-        razorpayUtils.placeOrder(onSuccess, onUserClose, beforeStart, '<?php echo $this->getFormId()?>', paymentIdField);
+        if (payment.currentMethod == '<?php echo $_code ?>') {
+            razorpayUtils.placeOrder(onSuccess, onUserClose, beforeStart, '<?php echo $this->getFormId()?>', paymentIdField);
+        } else {
+            review.save();
+        }
     };
 
     Event.observe(defaultPlaceOrderButton, 'click', placeOrder);
-</script>
+<?php else: ?>
+    var defaultPlaceOrderButton = $$('button[type=submit][class~="btn-checkout"]')[0];
+    defaultPlaceOrderButton.setAttribute('onclick', 'review.save();');
```

Comment:
Don't use onclick as the attribute. Use it as a handler. https://stackoverflow.com/questions/6941483/onclick-vs-event-handler